classes/class-wc-connect-nux.php

```diff
@@ -156,7 +156,28 @@ public function register_order_page_labels_pointer( $pointers ) {
 			return $pointers;
 		}
 
-		public static function get_banner_type_to_display( $status = array() ) {
+		/**
+		 * Check that the current user is the owner of the Jetpack connection
+		 * - Only that person can accept the TOS
+		 *
+		 * @return bool
+		 */
+		public function can_accept_tos() {
+			// Developer case
+			if ( defined( 'JETPACK_DEV_DEBUG' ) && JETPACK_DEV_DEBUG ) {
```

Comment:
We used to also check whether a user is an admin by using `current_user_can( 'manage_options' )`. Should we do that in this PR by adding it to `set_up_nux_notices()` on line 304 https://github.com/Automattic/woocommerce-services/blob/c01a35f7c1ec3a245a8dd86bf8beea1e366b91fc/classes/class-wc-connect-nux.php#L304 ?